### PR TITLE
Include response coverage in custom reports

### LINKED ISSUE #427: extend Swagger coverage recording to response coverage
Currently, webtau records coverage up to the operations level (method + path).

Many swagger specs contain response definitions for happy and error scenarios. It'd be great to report against coverage of each defined response (200, 400, 404, 500, ...).
I'd imagine this to be a separate coverage number from today's operations coverage, similar to method vs class coverage in unit tests.

### PR DESCRIPTION
**Pull request by tsiq-karold**
Fixes #427 

We currently present list of covered and non-covered operations.  This PR adds a separate data item which is covered and non covered responses for operations (e.g. spec defines 200 and default, are both of these covered?).

### CODE REVIEW

File: webtau-open-api/src/main/java/com/twosigma/webtau/openapi/OpenApiCoverage.java [hunk 1]
```diff
@@ -39,18 +42,103 @@ public void recordOperation(HttpValidationResult validationResult) {
...
+                                        Function.identity(),
+                                        resp -> 0))
+                        )
+                );
+
+        coveredOperations.getActualCalls().forEach(entry -> {
+            OpenApiOperation op = entry.getKey();
+            Set<Integer> actualResponses = entry.getValue().stream().map(OpenApiCoveredOperations.Call::getStatusCode).collect(Collectors.toSet());
+            Map<String, Integer> declaredResponses = operationToResponseToInvocationCount.get(op);
+
```
Comment (MykolaGolubyev) [inline webtau-open-api/src/main/java/com/twosigma/webtau/openapi/OpenApiCoverage.java]: extra empty line?

**Comment by tsiq-karold** [comment]
> This is pretty good! Is it exposed on report automatically?

Depends on what you mean by automatically 😄 The changes I made to `OpenApiReportDataProvider` expose it to custom reports, assuming I've understood the code correctly, I plan on testing it from our webtau tests before hitting merge.

**Comment by MykolaGolubyev** [comment]
I meant I can't remember if HTML report shows whatever key values there are, or there is an explicit code to expose some of the values.

**Comment by tsiq-karold** [comment]
Oh , I see.  I'll check the html report.

**Pull request merged**